### PR TITLE
Add validation loss to log

### PR DESCRIPTION
**Pull request by Frank995**
I think it would be good to also print the validation loss after each epoch to monitor overfitting.
I'm not an expert of tqdm and I wasn't able to change the description bar after the for loop, but I left the code anyway.

### CODE REVIEW

**Comment by cschaefer26** [comment]
Hi, thanks for the PR. The validation loss is already implemented, here:
https://github.com/as-ideas/DeepPhonemizer/blob/b8f170764c7648fe2acb552b787099ab4f941e58/dp/training/trainer.py#L130
